packages/react/src/components/ImageWithCaption/__stories__/ImageWithCaption.stories.js

```diff
@@ -52,7 +52,12 @@ export const Default = () => {
       })}>
       <div className="bx--row">
         <div className="bx--col-sm-4 bx--col-lg-8 bx--offset-lg-4">
-          <ImageWithCaption inverse={inverse} image={image} heading={heading} />
+          <ImageWithCaption
+            inverse={inverse}
+            image={image}
+            heading={heading}
+            lightbox={boolean('lightbox', true)}
```

Comment:
@annawen1 I think you're missing the `copy` prop here

Reply:
ah yep, I actually don't know where the `copy` text is supposed to go visually

@wonilsuhibm @larahanlon2 I see in the [func specs](https://ibm.ent.box.com/file/640289462720) in the `Caption` section that in the @sm breakpoint we show both `heading` (the caption that we have currently showing) and `copy` however I don't see how the `copy` is supposed to appear in the [visual specs](https://ibm.ent.box.com/file/640317162531) (there aren't any mobile specs). Can you provide clarification on this?

Reply:
Hey @annawen1 I took a look at the `image with caption` and `lightbox` specs. I'm a little confused about the `copy` prop. For `image with caption` it seems that it should be an option in addition to the `heading` but I'm not seeing any visual spec for that, like you. But I also wonder why the copy would be included in `image with caption` component. Is it different to `lightbox` description field or the same just a different style? I can check with Lila otherwise we may need to wait for Wonil.

Reply:
Is the copy going into the lightbox?

Reply:
yep the copy goes into lightbox. BUT according to the func specs, the lightbox functionality is only available for md to larger breakpoints. In mobile breakpoint, the user doesn't have the option launch the lightbox and they instead see the image, heading, and copy.